addons/web/static/src/search/search_model.js

```diff
@@ -939,10 +940,97 @@ export class SearchModel extends EventBus {
         this._notify();
     }
 
+    /**
+     * Because it require a RPC call to get the properties search items,
+     * it's done lazily, only when we need them.
```

Comment:
That's not really what the method is about, I would put this docstring on the caller instead.